mne/io/fiff/tests/test_raw_fiff.py

```diff
@@ -799,6 +799,14 @@ def test_crop():
         assert_equal(raw.last_samp, ar.last_samp)
         assert_array_equal(raw[:, :][0], ar[:, :][0])
 
+    # test shape consistency of cropped raw
+    data = np.zeros((1, 1002001))
+    info = create_info(['1'], 1000)
+    raw = RawArray(data, info)
+    for tmin in np.arange(0, 1001, 100):
```

Comment:
FYI `range` should be a little bit better on Py3k because it's an iterator. I suspect it might even be faster in Py27. Any reason to use `arange` here instead of `range`? Not a blocker for merge but I'm curious.